app/controllers/thredded/messageboard_groups_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Thredded
+  class MessageboardGroupsController < Thredded::ApplicationController
+
+    def new
+      @messageboard_group = MessageboardGroup.new
+      authorize_creating @messageboard_group
```

Comment:
`authorize_creating` will try to throw a custom error class that doesn't exist. We don't really need a custom exception class here, so let's just use `authorize @messageboard_group, :create?`